plugin/pom.xml

```diff
@@ -25,7 +25,7 @@
     <changelist>-SNAPSHOT</changelist>
     <module.name>${project.groupId}.warnings.ng</module.name>
 
-    <analysis-model-api.version>10.13.0</analysis-model-api.version>
+    <analysis-model-api.version>10.14.0-SNAPSHOT</analysis-model-api.version>
```

Comment:
This will not work in CI. You need to create another PR similar to 

https://github.com/jenkinsci/analysis-model-api-plugin/pull/61/files

This PR will create an incremental release that needs to reference your changes in https://github.com/jenkinsci/analysis-model/pull/813.

The incremental version will be available if all checks are green.